src/Repository/ProgrammeRepository.php

```diff
@@ -114,4 +114,22 @@ public function getPaginatedFilteredSorted(
 
         return $query->getQuery()->getResult();
     }
+
+    public function removeTrainerById(int $id): void
```

Comment:
try to find a more specific name here, something like `removeTrainersWithIdFromProgrammes`